case_studies/coadds/align_single_exposures.py

```diff
@@ -0,0 +1,39 @@
+# galsim_decoder changes for coadds
```

Comment:
could you add a minimal notebook showing the `align_single_exposures` functions work with a couple of images?  You probalby already showed that in the meeting sometime and just have to remove some extra stuff. Thank you :)